### PR TITLE
Support to handle Subscriber add code in HSS

### PR DESCRIPTION
**Pull request by thakurajayL**
Support to handle Subscriber add code in HSS. If any config is missing then request for sending new config

### CODE REVIEW

**Comment by yoooou** [comment]
test container

**Comment by yoooou** [comment]
retest this please

**Comment by yoooou** [comment]
retest this please

**Review by badhrinathpa — APPROVED** [review]
+1

File: hss/src/resthandler.cpp [hunk 1]
```diff
@@ -21,9 +21,156 @@ void RestHandler::onRequest(const Pistache::Http::Request& request, Pistache::Ht
...
    std::cout << request.method() << std::endl;
    std::cout << request.body() << std::endl;
+   static bool needConfig = true;
 
-   if (request.resource() == "/v1/config/imsis")
-   {
+   if (request.resource() == "/v2/config-check") {
+     if (needConfig == true) {
+        response.send(Pistache::Http::Code::Not_Found);
+        needConfig = false;
```
Comment (badhrinathpa) [inline hss/src/resthandler.cpp:29]: should we set needConfig to false after getting config??

Comment (thakurajayL) [reply hss/src/resthandler.cpp:29]: Since HSS is per subscriber, I am doing it as soon as config requested.  and not when we really receive configuration.

**Review by badhrinathpa — APPROVED** [review]
+1

**Pull request merged**